Frontend/src/Components/Graph/CustomGraph.js

```diff
@@ -63,18 +59,23 @@ for (const category of GraphData.Output) {
   }
 }
 
+
 // the options fed into the graph object, save regardless of datasets
 function getOptions(now, secondsRetained, colorMode) {
   // console.log("now:", now);
   // console.log("diff:", DateTime.now().diff(now).toHuman());
 
+  /**
+ * Stores all the units in the graph-data json file as key: value pairs
+ */
+
   const gridColor = getColor("grid", colorMode);
   const gridBorderColor = getColor("gridBorder", colorMode);
   const ticksColor = getColor("ticks", colorMode);
 
   return {
     responsive: true,
-    maintainAspectRatio: false,
+    maintainAspectRatio: true,
```

Comment:
```suggestion
    maintainAspectRatio: false,
```
This is probably your issue with the graphs not fitting vertically